scripts/xnat/check_new_sessions

```diff
@@ -319,30 +319,36 @@ def incomplete_scan_check(eid, manufacturer, label):
     missing_scans = []
     sess_type = 'xnat:mrSessionData/scans/scan/type'
     scan_types = ifc.select.experiment(eid).attrs.get(sess_type)
+    scan_types_set = set(scan_types)
+    ge_scan_types = set(scan_type_dictionary['GE MEDICAL SYSTEMS'])
+    siemens_scan_types = set(scan_type_dictionary['SIEMENS'])
+
     if manufacturer == 'GE MEDICAL SYSTEMS':
-        for scantype in set(scan_type_dictionary['GE MEDICAL SYSTEMS']):
-            if scantype in set(scan_types):
-                scantype = scantype
-            else:
-                missing_scans.append(scan_types)
+        if ge_scan_types.issubset(scan_types_set):
+            return
```

Comment:
I have tried and as far as I understand the problematic, `issubset()` is the right approach. 

s.issubset(t) 	s <= t 	test whether every element in s is in t
s.issuperset(t) 	s >= t 	test whether every element in t is in s

If we use `issuperset` we would only reassure that the scan_types_set are part of GE set, not knowing if there are missing ones.